### PR TITLE
RCv3 bulk add success semantics

### LINKED ISSUE #1082: Regain error recovery for RCv3 add/delete in worker code
Currently, these things get the raw result (`response, body`). That's fine, except that the success_pred is now `has_code(20x, 409)`, because there's supposed to be a callback that parses 409s in order to recover; at least in convergence.

This means that 409s are now considered success, which is clearly wrong.

Perhaps the resolution is to run the callback, check if the result is `Success, []`, and, if so, return the body; otherwise, raise an exception?

### PR DESCRIPTION
**Pull request by lvh**
This covers the case where a machine has already been added. This should probably also only happen after I resolve #1082 with #1088, otherwise we might break RCv3.

### CODE REVIEW

File: otter/test/convergence/test_steps.py [hunk 1]
```diff
@@ -494,6 +516,83 @@ def test_no_such_lb_message(self):
...
+        (partial_check_bulk_add, _), = eff.callbacks
+        self.assertEqual(partial_check_bulk_add.func,
+                         _rcv3_check_bulk_add)
+        expected_pairs = pset([(lb_b_id, node_b_id)])
+        self.assertEqual(partial_check_bulk_add.args, (expected_pairs,))
+        self.assertEqual(partial_check_bulk_add.keywords, None)
+
+    def test_node_already_a_member(self):
+        """
+        If the node was already a member, the request is successful.
```
Comment (cyli) [inline otter/test/convergence/test_steps.py]: Non-blocking nitpick:  to clarify the distinction between this test and the previous test, maybe "if all nodes are already members"?

**Comment by cyli** [comment]
LGTM if travis unwedges itself and the tests pass :)

**Comment by cyli** [comment]
Oops, looks like the worker tests had a test for `has_code(201)`, which probably needs to be updated to `has_code(201, 409)` ?

**Comment by lvh** [comment]
jenkins please run integration tests

**Comment by lvh** [comment]
jenkins please run integration tests

**Comment by lvh** [comment]
jenkins please run integration tests

**Comment by lvh** [comment]
I am ignoring coveralls since it's still wrong; it's complaining about a bunch of code I didn't even touch.

**Comment by lvh** [comment]
jenkins please run integration tests

**Comment by radix** [comment]
jenkins please run integration tests

**Comment by cyli** [comment]
jenkins please run integration tests

**Comment by cyli** [comment]
jenkins please run integration tests

**Pull request merged**